src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/choice_widget_options.html.php

```diff
@@ -8,6 +8,6 @@
             <?php echo $formHelper->block($form, 'choice_widget_options', array('choices' => $choice)) ?>
         </optgroup>
     <?php else: ?>
-        <option value="<?php echo $view->escape($choice->value) ?>" <?php echo $view['form']->block($form, 'attributes', array('attr' => $choice->attr)) ?><?php if ($is_selected($choice->value, $value)): ?> selected="selected"<?php endif?>><?php echo $view->escape($translatorHelper->trans($choice->label, array(), $translation_domain)) ?></option>
+        <option value="<?php echo $view->escape($choice->value) ?>" <?php echo $view['form']->block($form, 'attributes', array('attr' => $choice->attr)) ?><?php if ($is_selected($choice->value, $value)): ?> selected="selected"<?php endif?>><?php echo $view->escape($choice_translation_domain ? $translatorHelper->trans($choice->label, array(), $translation_domain) : $choice->label) ?></option>
```

Comment:
this is wrong when domain is null. would be better to reverse logic and use `false === domain` as in twig